meta-luneos/recipes-connectivity/mobile-broadband-provider-info/mobile-broadband-provider-info_git.bbappend

```diff
@@ -1,11 +1,18 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
+
 #Mer uses their own version of MBPI which has some elements added. So we want to use their version only in case we're using their oFono version. In other cases where we use upstream oFono we want to use the regular MBPI from Yocto.
 
 SRC_URI:halium  = " \
                     git://github.com/sailfishos/mobile-broadband-provider-info.git;protocol=https;branch=master \
                     file://meson.build \
```

Comment:
You can probably use subdir=git/${BPN} here to avoid the need for do_configure:prepend

Reply:
ah, right, that's more elegant. Let me add that.